### PR TITLE
StyleClass: Add .rounded style constant

### LINKED ISSUE #263: Add "rounded" as style constant
I thought this was a constant, but it looks like it's just a defacto class?

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/69472238-add-rounded-as-style-constant?utm_campaign=plugin&utm_content=tracker%2F60830613&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F60830613&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by cassidyjames**
Fixes #263. Usage in the demo is only supported in greenfield stylesheet.

### CODE REVIEW

**Comment by cassidyjames** [comment]
Probably need to bump the version and add a note to the release description?

File: demo/Views/CSSView.vala [hunk 1]
```diff
@@ -44,12 +44,16 @@ public class CSSView : Gtk.Grid {
...
 
-        var card_label = new Gtk.Label ("\"card\" style class:");
+        var card_label = new Gtk.Label ("\"card\" with \"rounded\" style class:");
         card_label.halign = Gtk.Align.END;
 
         var card = new Gtk.Grid ();
         card.orientation = Gtk.Orientation.VERTICAL;
-        card.get_style_context ().add_class (Granite.STYLE_CLASS_CARD);
+
+        var card_context = card.get_style_context ();
```
Comment (danrabbit) [inline demo/Views/CSSView.vala:53]:
```suggestion
        unowned Gtk.StyleContext card_context = card.get_style_context ();
```

File: meson.build [hunk 2]
```diff
@@ -2,7 +2,7 @@ project(
     'granite',
     'vala', 'c',
     meson_version: '>= 0.48.2',
-    version: '5.4.0'
+    version: '5.4.1'
```
Comment (danrabbit) [inline meson.build:5]: probably minor not patch right?
```suggestion
    version: '5.5.0'
```

Comment (cassidyjames) [reply meson.build:5]: ah yeah derp

File: data/granite.appdata.xml.in [hunk 3]
```diff
@@ -10,6 +10,14 @@
     <p>A companion library for GTK and GLib that provides complex widgets and convenience methods designed for use in apps built for elementary OS.</p>
   </description>
   <releases>
+    <release version="5.4.1" date="2020-05-07" urgency="medium">
```
Comment (cassidyjames) [inline data/granite.appdata.xml.in:13]:
```suggestion
    <release version="5.5.0" date="2020-05-07" urgency="medium">
```

**Pull request merged**